### PR TITLE
fix: Docs build

### PR DESCRIPTION
**Pull request by marksweb**
This updates the docs a bit, adds the same theme as django-cms & ensures bumpversion will keep the version update with releases

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -40,7 +44,8 @@
     url='http://github.com/ojii/django-classy-tags',
```
Comment (vinitkumar) [inline setup.py:44]: Please add the django-cms repo name, it still points to the old repo link of ojji

**Review by vinitkumar — CHANGES_REQUESTED** [review]
I looked into the code and made some comments. Please check the code and fix/resolve those comments.

File: setup.py (hunk 1 above)
Comment (marksweb) [reply setup.py:44]: Good spot. I thought I did that with the release last week

**Comment by marksweb** [comment]
@vinitkumar I've updated that url 👍

**Review by vinitkumar — APPROVED** [review]

**Pull request merged**